values.yaml

```diff
@@ -634,6 +634,12 @@ gitSyncRelay:
       gitlab.com ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBFSMqzJeV9rUzU4kWitGjeR4PWSa29SPqJ1fVkhtj3Hw9xjLVXVYrU9QlYWrOLXBpQ6KWjbjTDTdDkoohFzgbEY=
       gitlab.com ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIAfuCHKVTjquxvt6CM6tdG4SLp1Btn/nOeHHE5UOzRdf
   extraContainers: []
+  ## Enabling volumeSync disables the git-daemon and uses a shared volume as the sync mechanism between all airflow components and
+  ## the git-sync-relay
+  volumeSync:
+    volumeSize: 10Gi
+    # The specified storageClassName must support ReadWriteMany
```

Comment:
ReadWriteMany should this comment be removed in here

Reply:
That is a hint to the user. Actually, we have not been following the recommendations for documentation in helm charts, so maybe this is something we should improve? https://v2.helm.sh/docs/chart_best_practices/#document-values-yaml

Reply:
shall we remove the hard requirement of ```# The specified storageClassName must support ReadWriteMany```